ctx.py

```diff
@@ -13,20 +13,50 @@
 # limitations under the License.
 
 import functools
+import os
 
+from pydash import get
+import yaml
+
+import util
 '''
 Execution context. Filled upon invocation of cli.py, read by submodules
 '''
 
 args=None # the parsed command line arguments
 
+# TODO: Maybe provide a map name->path ?
+config=None
+
+# Load $HOME/.cc-utils.cfg if it exists
+if 'HOME' in os.environ:
+    util_config_path = os.path.join(os.environ['HOME'], '.cc-utils.cfg')
+    try:
+        with open(util_config_path) as config_file:
+            config = yaml.load(config_file.read())
+    except FileNotFoundError:
+        # could not find file - fail silently
+        pass
+    except Exception as e:
+        util.verbose(str(e))
+
+def get_config(path: str):
+    return get(config, path)
+
 def _cfg_factory_from_dir():
+
+    if get_config('ctx.cc-config_dir'):
+        util.verbose('using cfg_dir specified in cc-utils.cfg')
+        cfg_dir = get_config('ctx.cc-config_dir')
```

Comment:
expose the "ctx" dictionary via a separate getter from the above-mentioned configuration class; merge it with the argparse namespace objects (using util.merge_dicts); rename the attributes to match those defined in cli.py.add_global_args